cpp/ql/test/library-tests/permissive/calls.expected

```diff
@@ -1,2 +1 @@
-| non_permissive.cpp:6:3:6:3 | call to f | non_permissive.cpp:2:13:2:13 | f |
 | permissive.cpp:6:3:6:3 | call to f | permissive.cpp:2:13:2:13 | f |
```

Comment:
Did we discuss this change? It's likely we have, but I've forgotten about it.

Reply:
This was commented on here: https://github.com/github/codeql-c-extractor-team/issues/7#issuecomment-574109878 and the slack discussion here: https://github.slack.com/archives/CP75PA8Q1/p1578929814013400

Reply:
Thanks. In conclusion, the result is still there when mimicking `-fpermissive`, which is the important thing.